.travis.yml

```diff
@@ -28,13 +40,13 @@ before_script:
 
   - composer validate
   - php bin/configure --default
-  - composer update --no-dev
+  - COMPOSER_MEMORY_LIMIT=-1 composer install --optimize-autoloader
```

Comment:
Removed `--no-dev` as PHPUnit is a dev dependency

Changed `update` to `install --optimize-autoloader` for consistency with the installation docs